src/Notification/index.scss

```diff
@@ -0,0 +1,79 @@
+@import './../shared';
+
+.alert {
+  @include card-overlay;
+
+  margin-top: $spacing-m;
+  padding: $spacing-s $spacing-m;
+
+  &.alert--action {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    @include media($until-medium-screens) {
+      flex-direction: row;
+    }
+  }
+
+  &.alert--error {
+    border-top: $spacing-xxs solid $color-error;
+  }
+
+  &.alert--info {
+    border-top: $spacing-xxs solid $color-brand;
+  }
+
+  &.alert--success {
+    border-top: $spacing-xxs solid $color-highlight;
+  }
+
+  &.alert--warning {
+    border-top: $spacing-xxs solid $color-warning;
+  }
+
+  &.alert--single-line {
+    p {
+      margin: 0;
+    }
+  }
+
+  &.alert--bank-wire-details {
+    @include media($until-medium-screens) {
+      display: flex;
+      width: 100%;
+
+      .alert--bank-wire-header {
+        width: 66%;
+      }
+
+      .alert--bank-wire-details-block {
+        width: 33%;
+      }
+    }
+  }
+
+  &.alert-doc-upload {
+    display: none;
+
+    @include media($until-big-screens) {
+      display: flex;
+    }
+  }
+}
```

Comment:
I don't think we need the `alert-doc-upload` or `alert--bank-wire-details`. Since you accept `className` as a prop, it's the user's responsibility to pass in those modifications :)